ui/docker/Dockerfile.build

```diff
@@ -0,0 +1,8 @@
+FROM node
```

Comment:
We'll need to rework the build infrastructure. This can be done in later PRs.

Reply:
I'm working on it. It will arrive in my next PR very soon.